app/concerns/controlled_keywords.rb

```diff
@@ -46,6 +46,17 @@ def set_science_keywords
     @science_keywords = keywords
   end
 
+  def fetch_keyword_recommendations
```

Comment:
I favor moving this to its own concern for GKR.  The recommendations are interacting with a controlled keyword field, but this is a different behavior than the rest of these.  It's not setting options.

Reply:
moved